### PR TITLE
add --quiet/-q flag and allow timing info to be turned on/off

### LINKED ISSUE #324: add timing toggle in datafusion cli to allow timing info printing to be turned on or off
**Is your feature request related to a problem or challenge? Please describe what you are trying to do.**

add timing toggle in datafusion cli to allow timing info printing to be turned on or off

**Describe the solution you'd like**

Add a new flag called `timing` and by default turned on, unless file a supplied in which case by default is off.

Either case users can override with the timing flag.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Jimexist**
# Which issue does this PR close?

add --quiet/-q flag and allow timing info to be turned on/off.

Closes #324

 # Rationale for this change

so that we can do better integration test

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: datafusion-cli/src/main.rs [hunk 1]
```diff
@@ -72,6 +69,14 @@ pub async fn main() {
                 .validator(is_valid_format)
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("timing")
```
Comment (alamb) [inline datafusion-cli/src/main.rs:73]: What do you think about making this option `quiet` or something like that and having it turn off all output (other than query results) and not just the timing information?

**Comment by alamb** [comment]
Thanks @Jimexist  this is looking very cool

File: datafusion-cli/src/main.rs (hunk 1 above)
Comment (Jimexist) [reply datafusion-cli/src/main.rs:73]: a good point - let me change to that

**Comment by Jimexist** [comment]
> Thanks @Jimexist this is looking very cool

changed to `quiet`

**Review by alamb — APPROVED** [review]
Looks good @Jimexist

**Review by Dandandan — APPROVED** [review]

**Pull request merged**